input/API_TEST_SRV.edmx

```diff
@@ -0,0 +1,331 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Is this used anywhere, we should rather extend the existing EDMX files.

Reply:
sorry commited by accident